google/auth/_default.py

```diff
@@ -172,7 +172,10 @@ def _get_explicit_environ_credentials():
 
 def _get_gae_credentials():
     """Gets Google App Engine App Identity credentials and project ID."""
-    from google.auth import app_engine
+    try:
+        from google.auth import app_engine
```

Comment:
This case is odd enough to warrant a comment explaining (to our future selves) how the import could possibly fail.

Reply:
Done